CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.20.0-dev
+* Feature - Support Docker volume and driver plugins.
```

Comment:
Probably it's better to list them as two items?
```
Feature - Add support for Docker volume plugins
Enhancement - Replace the empty container with Docker local volume
```

Reply:
We should add a note about locking out older docker versions.

Reply:
I' added deprecating older docker versions as enhancement.